src/core/utils/format-number.ts

```diff
@@ -50,3 +50,22 @@ export const formatNumber = (amount: number | BigNumber, options: INumberFormatO
 
 const integerRegex = new RegExp(/^\d+$/);
 export const isInteger = (value: string) => integerRegex.test(value);
+
+export const qrCodeRegex = /^mooonletExtSync:([^\@]*)\@([^\/\?]*)([^\?]*)?\??(.*)/;
```

Comment:
these stuff doesn't belong here :)
qrCode stuff can be moved where connection is done, it's specific to that feature